appengine/go11x/tasks/create_http_task/create_http_task.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START cloud_tasks_create_http_task]
+
+// Command create_task constructs and adds a task to an App Engine Queue.
+package main
```

Comment:
Generally, snippets should not be in a package main. From the sample rubric, snippets should not be runnable unless they're a quickstart because we've had issues maintaining them. Tests should ensure the snippet works.

Reply:
This is a quickstart, but if it wouldn't be confusing to run it as a test I'd be open to working with someone to change it over.